src/salaryAndBenefits/utils/salaryData.ts

```diff
@@ -0,0 +1,41 @@
+export const payscale = {
+    '2024': {
+      "2024" : "600000",
```

Comment:
Still think the salary values should be numbers

Reply:
It depends on what we add once a year I believe, we might want to adjust the script.